rtsp.go

```diff
@@ -30,12 +30,30 @@ import (
 )
 
 var family = resource.ModelNamespace("erh").WithFamily("viamrtsp")
+var ModelAgnostic = family.WithModel("rtsp")
 var ModelH264 = family.WithModel("rtsp-h264")
+var ModelH265 = family.WithModel("rtsp-h265")
+var Models = []resource.Model{ModelAgnostic, ModelH264, ModelH265}
+
+func modelToCodec(model resource.Model) videoCodec {
+	switch model {
+	case ModelAgnostic:
+		return Unknown
+	case ModelH264:
+		return H264
+	case ModelH265:
+		return H265
+	default:
+		return Unknown
```

Comment:
I think this case should be an error. So you will want to change the signature of this function to return an `(videoCodec, err)`

Reply:
done